packages/app/src/systems/SignUp/components/WalletCreated/components/PinWalletCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { cssObj } from '@fuel-ui/css';
```

Comment:
about `PinWalletCard` 

- can we move it to `systems/Core` ?
- can we create a simple story for it?

Reply:
It doesn't need to be in the core, as it's not supposed to be used elsewhere. About the storybook will create it.